app/models/claim.rb

```diff
@@ -121,11 +121,11 @@ def can_claim_youth_court?
     claim_type == ClaimType::NON_STANDARD_MAGISTRATE.to_s &&
       !before_youth_court_cutoff? &&
       youth_court == 'yes' &&
-      plea_category.match(/category_[12]a/)
+      plea_category.match(/category_[12]a/).present?
```

Comment:
`.match?`

Reply:
I just removed it, you're right - it's not really necessary to do

Reply:
(as in removed .present?)